plugins/lime-plugin-network-admin/src/netAdminMenu.js

```diff
@@ -3,5 +3,8 @@ import { h } from 'preact';
 import { Trans } from '@lingui/macro';
 
 export const NetAdminMenu = () => (
-	<a href='#/netadmin'><Trans>Network Configuration</Trans></a>
+	<span>
+		<svg viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg" width="15" height="15"><path d="M4.5 6.5v-3a3 3 0 016 0v3m-8 0h10a1 1 0 011 1v6a1 1 0 01-1 1h-10a1 1 0 01-1-1v-6a1 1 0 011-1z" stroke="currentColor" /></svg>
+		<a href='#/netadmin'><Trans>Shared Password</Trans></a>
```

Comment:
Oops, this change shouldn't have gone. But this section really should change it's name as the only thing it does for now is change password.

Reply:
Hi Luandro!
Thanks for this...
I would invite you to propose that change in another PR.